src/main/java/wansbot/ui/UI.java

```diff
@@ -0,0 +1,247 @@
+package wansbot.ui;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import wansbot.tasks.Task;
+import wansbot.tasks.TaskList;
+
+/**
+ * This class handles all the UI that will be outputted to user.
+ */
+public class UI {
+    private static final String HR = "----------------------------------------------------------------------";
+
+    public UI() {
+    }
+
+    /**
+     * Method that displays Bot message when bot first starts.
+     */
+    public void introduceToUser() {
+        String logo = "                 __"
+                + "\n|  |  /\\  |\\ | /__` "
+                + "\n|/\\| /~~\\ | \\| .__/\n";
+
+        System.out.println(HR + "\nWans:\n"
+                + "Hey, I'm\n"
+                + logo
+                + "\nCan I help? (I can only manage a todo list so...)\n" + HR);
+    }
```

Comment:
I like how you gave your bot more personality

Reply:
Just trying to make this long exercise a little more fun. :o